app/components/insert-title-card.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import insertTitle from '../commands/insert-title';
+import { inject as service } from '@ember/service';
+
+export default class InsertTitleCardComponent extends Component {
+  @service intl;
+
+  @action
+  insertTitle() {
+    this.args.controller.doCommand(insertTitle(this.intl), {
+      view: this.args.controller.mainEditorView,
+    });
+    //this.args.controller.focus();
+  }
+
+  canInsertTitle = () =>
```

Comment:
why is this not a getter? I'm seeing this style pop up across PRs from 3 different people, did I miss a memo?
(genuine question, not rhetorical :sweat_smile: )